### ISSUE TITLE
toString() produces cards with `\n`, not `\r\n`

### ISSUE THREAD

**Issue by bengotow** [jhermsmeier/node-vcf#33]
Code to reproduce:

```
let vcf = <a card>

// Parse into vCard
const card = new vCard().parse(vcf);

// Serialize back to string
vcf = card.toString()

// Throws an exception because newlines in new vcf string are `\n` and the parser expects `\r\n`
const card = new vCard().parse(vcf);

**Comment by jhermsmeier** [comment]
Thanks! That was an embarassing bug 😄 
Fix published in `vcf@2.0.6`

**Comment by bengotow** [comment]
Wow thanks for the quick turnaround, that's awesome! 🙏
